src/connectors/youtube-music.js

```diff
@@ -23,10 +20,7 @@ Connector.getTrackArt = () => {
 	return null;
 };
 
-Connector.albumSelector = [
-	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/MPREb_"]',
-	'.ytmusic-player-bar .yt-formatted-string.style-scope.yt-simple-endpoint[href*="browse/FEmusic_library_privately_owned_release_detailb_"]',
-];
+Connector.albumSelector = '.ytmusic-player-bar.byline .yt-formatted-string:nth-child(3)';
```

Comment:
These selectors don't work properly with multiple artists.
<details>
  <summary><b>Screenshot </b> (Click to expand)</summary>

![image](https://user-images.githubusercontent.com/1119267/109980362-e0d5f880-7d10-11eb-9d1f-3f1b7d3f6c07.png)

</details>

Reply:
Ah yes, I don't have any music with multiple artists so didn't realize they split up the names like that.

Guess the href filter would be required then. Problem is the new urls for album both start with with /channel, so that's annoying unless jquery allows something like negative forward searches so artist can be /channel/(?!=uMPREb)